packages/react-jss/src/index.d.ts

```diff
@@ -51,9 +53,28 @@ interface Options extends StyleSheetFactoryOptions {
 
 type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
 
+export {Styles}
+
+declare function createUseStyles<C extends string>(
+  styles: Record<C, Style>,
```

Comment:
The inference here doesn't work yet (Not sure why though, at least in Webstorm I couldn't get it working)